src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx

```diff
@@ -0,0 +1,274 @@
+import { cropped } from "Utils/resized"
+
+export type Specialty =
+  | "auctions"
+  | "priveteSalesAndAdvisory"
+  | "collectorServices"
+
+export interface SpecialistsData {
+  specialty: Specialty
+  image: { src: string; srcSet: string }
+  name: string
+  firstName: string
+  jobTitle: string
+  bio: string
+  email: string
+}
+
+const imageOptions = {
+  width: 644,
+  height: 445,
+}
+
+const TEST_imageAuctions = cropped(
+  "https://files.artsy.net/images/08_CVP_About_Follow (2).png",
+  imageOptions
+)
+
+const TEST_imagePriveteSalesAndAdvisory = cropped(
+  "https://files.artsy.net/images/06_CVP_About_Sell.png",
+  imageOptions
+)
+
+const TEST_CollectorServices = cropped(
```

Comment:
**question:** Is the `TEST_` prefix needed?

Reply:
I'm using it to point to the fact that these images are not final and should be replaced when we have actual photos of specialists